src/components/staking/PageValidator.vue

```diff
@@ -79,7 +79,11 @@
         <TmBtn
           id="undelegation-btn"
           class="undelegation-btn"
-          :disabled="delegation.amount === 0 && !isInactiveValidator"
+          :disabled="
+            delegation.amount === 0 &&
+            !isInactiveValidator &&
+            currentNetwork.network_type !== `polkadot`
```

Comment:
this doesn't work. case:
- polkadot
- none/controller account
- user has no delegation with a validator